pkg/acceptance/helpers/common_test.go

```diff
@@ -0,0 +1,46 @@
+package helpers
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMatchAllStringsInOrder(t *testing.T) {
+	testCases := map[string]struct {
+		parts     []string
+		text      string
+		wantMatch bool
+	}{
+		"empty parts and text": {
+			parts:     []string{},
+			text:      "",
+			wantMatch: true,
+		},
+		"empty parts": {
+			parts:     []string{},
+			text:      "xyz",
+			wantMatch: true,
+		},
+		"empty text": {
+			parts: []string{"a", "b"},
+			text:  "",
+		},
+		"matching non empty": {
+			parts:     []string{"a", "b"},
+			text:      "xyaxyb",
+			wantMatch: true,
+		},
+		"not matching": {
```

Comment:
nit: this is partial match, we can be explicit with it

Reply:
Fixed.